app/jobs/wakes/update_facebook_metrics_job.rb

```diff
@@ -2,7 +2,7 @@
 class Wakes::UpdateFacebookMetricsJob < ActiveJob::Base
   queue_as :wakes_facebook_metrics
 
-  def perform(locations)
+  def perform(*locations)
```

Comment:
So I missed seeing this before I squashed. There is no reason to leave it in, other than it’s harder to remove it now that it’s squashed :) and may be that if we call flatten in side the job, it can support `Wakes::UpdateFacebookMetricsJob.perform(location)`, `Wakes::UpdateFacebookMetricsJob
(locations)`, and `Wakes::UpdateFacebookMetricsJob.perform(*locations)`. Should I just add in `locations.flatten!` in the job? 


> On Mar 24, 2016, at 11:09 AM, Ben Hutton <notifications@github.com> wrote:
> 
> In app/jobs/wakes/update_facebook_metrics_job.rb <https://github.com/desiringgod/wakes/pull/16#discussion_r57341564>:
> 
> > @@ -2,7 +2,7 @@
> >  class Wakes::UpdateFacebookMetricsJob < ActiveJob::Base
> >    queue_as :wakes_facebook_metrics
> >  
> > -  def perform(locations)
> > +  def perform(*locations)
> @gaganawhad <https://github.com/gaganawhad> if not required, I think we remove, right? Can you see some reason to leave it in?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/desiringgod/wakes/pull/16/files/527a259df85f35b7d7035a427432bab5d65bae64..796c31829b486bfc8616a324d16246843c79310d#r57341564>